Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -110,3 +111,22 @@ public extension UIConfig {
         public var imageGalleryInteritemSpacing: CGFloat = 2
     }
 }
+
+// MARK: - MessageComposer
+
+public extension UIConfig {
+    struct MessageComposer {
+        public var messageComposerView: ChatChannelMessageComposerView<ExtraData>.Type =
+            ChatChannelMessageComposerView<ExtraData>.self
+        public var messageInputView: ChatChannelMessageInputView<ExtraData>.Type = ChatChannelMessageInputView<ExtraData>.self
+        public var attachmentsView: MessageComposerAttachmentsView<ExtraData>.Type = MessageComposerAttachmentsView<ExtraData>.self
+        public var sendButton: MessageComposerSendButton.Type = MessageComposerSendButton.self
+        public var composerButton: ChatSquareButton.Type = ChatSquareButton.self
+        public var colors: Colors = .init()
+        
+        public struct Colors {
+            public var composerButton: UIColor = .composerButton
+            public var composerStateIcon: UIColor = .composerStateIcon
```

Comment:
Why this indirection? Just put the colors here :)

Reply:
Also, do you think we should maybe introduce some kind of global "unselected" and "disabled" color?

Reply:
Color palette was handled by @ManWithBear so I've adopted usage of that in this PR.